Resources/Locale/en-US/holopad/holopad.ftl

```diff
@@ -123,6 +128,7 @@ holopad-security-lawyer = Security - Lawyer
 holopad-security-armory = Security - Armory
 holopad-security-locker-room = Security - Locker Room
 holopad-security-brig-med = Security - Brig Med
+holopad-security-checkpoint = Security - Checkpoint
```

Comment:
Some stations have so many security checkpoints a single "Security - Checkpoint" might give the wrong idea of adding it everywhere, thus having like 5 "Security - Checkpoint" per map with no means to differanciate them.
The better idea would be to just make a holopad specific to every checkpoint on every map but thatd be huge entity bloat